cobalt/watchdog/watchdog_test.cc

```diff
@@ -636,5 +636,14 @@ TEST_F(WatchdogTest, EvictOldWatchdogViolations) {
   ASSERT_EQ(watchdog_->GetWatchdogViolations({}, false), "");
 }
 
+TEST_F(WatchdogTest, CanGetLogTrace) {
```

Comment:
Can we flesh out some more tests here? E.g., I want to see the presence of log trace in a violation, the the public api methods are working as expected (eg. clearLog) etc.